book/website/community-handbook/infrastructure.md

```diff
@@ -0,0 +1,23 @@
+(ch-infrastructure)=
+# Infrastructure
+
+The Turing Way consists of many parts which need to work together to make sure the book can be build, looks correct and is available on the internet.
+There are processes to build, deploy and ensure the quality of the book.
+There is also configuration which may direct those processes or the site.
+Collectively we call these things infrastructure.
+
+Some examples include
+
+- Instructions to build the site from the source files
```

Comment:
This could link directly to this page? https://the-turing-way.netlify.app/community-handbook/local-build.html

Reply:
That's interesting. When I wrote this I was thinking about CI/CD specifically.

We could mention both here?

Reply:
Done in [62f6172](https://github.com/alan-turing-institute/the-turing-way/pull/3102/commits/62f617230069d5490029e9323a297ea12602f429)